iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -40,6 +40,70 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
     ASSERT_THAT(*instanceContainer.begin(), Eq(IdString("instance1")));
 }
 
+TEST_F(RoudiFindService_test, OfferServiceWithDefaultDescription)
```

Comment:
@nihalchari Please remember to question our behavior. The goal of the unit tests is not only to verify if something is working you should also verify if it makes sense that iceoryx behaves in this way. I think here you discovered a behavior which makes no sense, therefore it would make sense to correct it and verify it with a unit test.

- [ ] I think `offerService` should be declined when any of the following member values are set in the service description
```cpp
static constexpr uint16_t InvalidID = 0u;
static const IdString InvalidIDString{"0"};
static constexpr uint16_t AnyService = 0xFFFFu;
static constexpr uint16_t AnyInstance = 0xFFFFu;
static constexpr uint16_t AnyEvent = 0xFFFFu;
static constexpr char AnyServiceString[]{"65535"};
static constexpr char AnyInstanceString[]{"65535"};
static constexpr char AnyEventString[]{"65535"};
```
 - [ ] test the new `offerService` behavior with valid and invalid values

Reply:
okay , @elfenpiff  for adjusting offerService behavior not to allow invalid ServiceDescriptions, i will create a separate issue and link here, and adapt test OfferServiceWithDefaultDescription accordingly and disable for now, is it fine?

Reply:
@nihalchari This is fine by me.

Reply:
@elfenpiff  i created new issue https://github.com/eclipse/iceoryx/issues/359 and fixed other review points.
Could u please review